src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -413,12 +413,62 @@ private void setHighlights(Article article, ParsedPage page) {
 
 	}
 
+	/*
+	* Extracts text and links from tables and returns a list of paragraphs
+	* */
+	private List<Paragraph> getParagraphsInTables(ParsedPage page){
+		List<Paragraph> paragraphsInTables = new ArrayList<Paragraph>();
+		for(de.tudarmstadt.ukp.wikipedia.parser.Table t: page.getTables()){
```

Comment:
I would like to keep it like this.
There are Two `Link` and `Table` classes one from jsonpedia project and other from JWPL.
We use both in this file. Most of the time the one from jsonpedia